cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -121,6 +144,9 @@ func genService(gen *protogen.Plugin, file *protogen.File, g *protogen.Generated
 		g.P(deprecationComment)
 	}
 	g.P("func New", clientName, " (cc ", grpcPackage.Ident("ClientConnInterface"), ") ", clientName, " {")
+	if helper != nil {
```

Comment:
Can the helper be always set, with a default implementation which does the things that we do when `helper == nil`?

Reply:
Done. Removed check.